src/DocumentComponent.ts

```
@@ -0,0 +1,38 @@
+ */
+export class DocumentComponent extends NodeComponent<DocumentFragment> {
+    public constructor(html?: string) {
+        super(html ? document.createRange().createContextualFragment(html) : document.createDocumentFragment());
+    }
+
+    /**
+     * Template literal tag function that accepts HTML code with components in a
+     * string literal
+     */
+    public static tag(strings: TemplateStringsArray, ...components: NodeComponent<any>[]): DocumentComponent {
+        const idPrefix = "tag-" + crypto.randomUUID() + "-";
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)